packages/p2p/src/components/app.jsx

```diff
@@ -157,7 +157,9 @@ const App = observer(props => {
                             <Tabs
                                 onTabItemClick={general_store.handleTabClick}
                                 active_index={general_store.active_index}
-                                className='p2p-cashier'
+                                className={classNames('p2p-cashier', {
+                                    'p2p-cashier__advertiser-tab': is_mobile && general_store.is_advertiser,
+                                })}
```

Comment:
Why does the main P2P component get a child component's class name? 🤔

Reply:
Because changing the length of tab happened only we. have an advertiser-tab